src/sidebar/config/icons.js

```diff
@@ -0,0 +1,54 @@
+// @ts-nocheck - TS doesn't understand `require('.../icon.svg')`
+
+/**
+ * Set of icons used by the sidebar application via the `SvgIcon`
+ * component.
+ */
+export default {
```

Comment:
I don't think "icons" fit into the category of configuration. If we had a collection of modules like this I don't probably call them `resources`. As it stands I think it is a unique enough thing to have at the top level of the application.